### PR TITLE
BXMSDOC-2780 Release Notes for PAM 7.0 - update 3

### PR DESCRIPTION
**Pull request by emmurphy1**
Including new features from Ivo's comments.

### CODE REVIEW

File: docs/product-release-notes/src/main/asciidoc/rn-whats-new-con.adoc [hunk 1]
```diff
@@ -130,6 +135,17 @@ Initial integration with NoSQL has been introduced. This is based on additional
 === Quartz improvements
 The Quartz scheduler service has been improved to fetch jobs only for deployments (KIE containers) that are currently available in the running system. 
 
+=== SLA due date tracking
+You can now track your processes, the activities in your processes, and cases by service level agreement (SLA) due date.
+
+=== Email interaction to start a process or complete a task — ask Maciej for details
```
Comment (michelehaglund) [inline docs/product-release-notes/src/main/asciidoc/rn-whats-new-con.adoc]: Draft note remaining

File: docs/product-release-notes/src/main/asciidoc/rn-whats-new-con.adoc [hunk 2]
```diff
@@ -130,6 +135,17 @@ Initial integration with NoSQL has been introduced. This is based on additional
 === Quartz improvements
 The Quartz scheduler service has been improved to fetch jobs only for deployments (KIE containers) that are currently available in the running system. 
 
+=== SLA due date tracking
+You can now track your processes, the activities in your processes, and cases by service level agreement (SLA) due date.
```
Comment (michelehaglund) [inline docs/product-release-notes/src/main/asciidoc/rn-whats-new-con.adoc:144]: Should the SLA phrase have initial caps?

File: docs/product-release-notes/src/main/asciidoc/rn-whats-new-con.adoc [hunk 3]
```diff
@@ -130,6 +135,17 @@ Initial integration with NoSQL has been introduced. This is based on additional
...
+
+=== Email interaction to start a process or complete a task — ask Maciej for details
+
+=== Task enhancement
+You can now forward and reassign a task to groups.
+
+=== `AsyncMode` environment variable
+You can use the `AsyncMode` environment variable to set all tasks to be  asynchronous by default.
```
Comment (michelehaglund) [inline docs/product-release-notes/src/main/asciidoc/rn-whats-new-con.adoc]: extra space in before "asynchronous"

File: docs/product-release-notes/src/main/asciidoc/rn-known-issues-con.adoc [hunk 4]
```diff
@@ -67,7 +67,7 @@ Actual result: Business Central does not deploy and an error is recorded in the
 
 It is possible to embed designers to hide the path to projects. However, this feature is not working in the Authoring page if you use a parameter-controlled URL.
 
-.Project-0riented standalone pages are broken [https://issues.jboss.org/browse/RHPAM-503[RHPAM-503]]
+.Project-oriented standalone pages are broken [https://issues.jboss.org/browse/RHPAM-503[RHPAM-503]]
```
Comment (michelehaglund) [inline docs/product-release-notes/src/main/asciidoc/rn-known-issues-con.adoc:70]: Should there be a period after this sentence? Dunno :) Same may apply in remaining sections.

File: docs/product-release-notes/src/main/asciidoc/rn-known-issues-con.adoc [hunk 5]
```diff
@@ -20,7 +20,7 @@ Actual result: The first two instances that were opened perform as usual, but th
 
 Workaround: Set the value of `chrome://flags/#force-color-profile` to sRGB.
 
-.With Process Designer, ensure embedded sub-process have enough storage [https://issues.jboss.org/browse/RHPAM-426[RHPAM-426]]
+.With Process Designer, ensure embedded sub-processes have enough storage [https://issues.jboss.org/browse/RHPAM-426[RHPAM-426]]
```
Comment (michelehaglund) [inline docs/product-release-notes/src/main/asciidoc/rn-known-issues-con.adoc:23]: Should there be a period after this sentence? Dunno :)

File: docs/product-release-notes/src/main/asciidoc/rn-whats-new-con.adoc [hunk 6]
```diff
@@ -83,6 +83,9 @@ The deployment descriptor designer performs the validation when a build is invok
 === Job creation
 The Job creation interface has been modified to enable you to start the job immediately or schedule it to be executed later.
 
+=== Application templates
+You can use the Employee Roster and Case Management app as application templates.
```
Comment (michelehaglund) [inline docs/product-release-notes/src/main/asciidoc/rn-whats-new-con.adoc]: I think this is OK as is, but do we use "app" in our customer docs?

File: docs/product-release-notes/src/main/asciidoc/rn-whats-new-con.adoc (hunk 2 above)
Comment (emmurphy1) [reply docs/product-release-notes/src/main/asciidoc/rn-whats-new-con.adoc:144]: No. On cap spelled-out acronyms if they are proper nouns. @michelehaglund

**Pull request merged**